Clarify `req.originalUrl` behaviour with query string in request (#1231)
expressjs/expressjs.com: _includes/api/en/4x/req-originalUrl.md

// => '/search?q=something'
```

> `req.originalUrl` available both in middleware and router objects. It is a
> combination of `req.baseUrl` and `req.path` values. It will also include query
> string in case it will be present in request. Consider following example:

Reviewer: Just some minor proofreading/language smoothing:

```suggestion
`req.originalUrl` is available both in middleware and router objects. It is a
combination of `req.baseUrl`, `req.path`, and any query string included
in the request. Consider following example:
```

Reviewer 2: Maybe we should really just say that it is the combination of `req.baseUrl` and `req.url`? I think, looking back at the issue, this is probably how it should have been specified, as `req.path` is just `req.url` without the query string.
